nautobot/core/api/urls.py

```diff
@@ -36,4 +38,13 @@
     path("graphql/", GraphQLDRFAPIView.as_view(), name="graphql-api"),
     # Plugins
     path("plugins/", include((plugin_api_patterns, "plugins-api"))),
+    # Lookup Expr
+    path(
+        "filterset-fields/lookup-choices/", GetFilterSetFieldLookupExpressionChoicesAPI.as_view(), name="lookup_choices"
+    ),
+    path(
+        "filterset-fields/lookup-value-dom-element/",
+        GetFilterSetFieldDOMElementAPI.as_view(),
+        name="lookup_value_dom_element",
+    ),
```

Comment:
Yeah, it's awkward since core's view URLs and API URLs haven't been namespaced to date. It probably is the right direction to move for new URL patterns in core, though.

Reply:
Also, the names for these views should ideally be namespaced as well, something like `core-api:filtersetfield-list-lookupchoices` and `core-api:filtersetfield-retrieve-lookupvaluedomelement`.